src/Reference.php

```diff
@@ -196,7 +196,6 @@ protected function initTableAlias(): void
 
             $this->table_alias = $this->link;
             $this->table_alias = preg_replace('~_(' . preg_quote($ourModel->id_field, '~') . '|id)$~', '', $this->table_alias);
-            $this->table_alias = preg_replace('~([a-z])[a-z]*[^a-z]*~i', '$1', $this->table_alias);
```

Comment:
@DarkSide666 @georgehristov what is the purpose of this regex replace? - one some MSSQL test is failing

plus for some reasons, stack trace is not shown in failing test with PHP 8:
```
1) Atk4\Data\Tests\SubTypesTest::testBasic
Undefined array key ":Ob"

/__w/data/data/vendor/atk4/dsql/src/Mssql/ExpressionTrait.php:48
/__w/data/data/vendor/atk4/dsql/src/Mssql/ExpressionTrait.php:52
/__w/data/data/vendor/atk4/dsql/src/Expression.php:656
/__w/data/data/src/Persistence/Sql.php:683
/__w/data/data/src/Persistence/Sql.php:714
/__w/data/data/src/Model.php:1131
/__w/data/data/src/Model.php:1156
/__w/data/data/src/Model.php:1578
/__w/data/data/vendor/atk4/dsql/src/Connection.php:365
/__w/data/data/src/Persistence/Sql.php:132
/__w/data/data/src/Model.php:1881
/__w/data/data/src/Model.php:1592
/__w/data/data/tests/SubTypesTest.php:34
/__w/data/data/tests/SubTypesTest.php:151
```